stats.js

```diff
@@ -0,0 +1,207 @@
+const apiKey = 'zQOIBdFR8M0sYuT0njD9ku8oL'; // Socrata API Key.
```

Comment:
Can we pull this from the config/environment?

Reply:
Done